app/helpers/application_helper.rb

```diff
@@ -1,2 +1,4 @@
 module ApplicationHelper
```

Comment:
Please do not push empty files like this one.
File should only be pushed if it is being worked on.